librz/include/rz_type.h

```diff
@@ -33,21 +33,29 @@ typedef struct rz_type_t {
 	RzIOBind iob; // for RzIO in formats
 } RzTypeDB;
 
+// All types in RzTypeDB module are either concrete,
+// "base" types that are types already having the
+// concrete size and memory layout
+// or the "AST" types that are returned from the parser
+// and don't contain the size or memory laoyout
+
+// Base types
+
 typedef struct rz_type_enum_case_t {
-	char *name;
+	const char *name;
```

Comment:
Is this really correct? Iirc the RzTypeEnumCase and others own these strings.

Reply:
Fixed.